---
source: tooling/nargo_cli/tests/execute.rs
expression: "&format!(\"success: {:?}\\nexit_code: {}\\n----- stdout -----\\n{}\\n----- stderr -----\\n{}\",\noutput.status.success(), output.status.code().unwrap_or(!0), stdout, stderr)"
---
success: false
exit_code: 1
----- stdout -----

----- stderr -----
error: 2 + 3 != 6
  ┌─ src/main.nr:6:5
  │
6 │     std::static_assert(xy == 6, "2 + 3 != 6");
  │     -----------------------------------------
  │
  = Call stack:
    1. src/main.nr:6:5

Aborting due to 1 previous error
